autoload/coverage/python.vim

```diff
@@ -0,0 +1,79 @@
+" Copyright 2014 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+"{{{ Init
+
+let s:plugin = maktaba#plugin#Get('coverage')
+
+"}}}
+
+"{{{ coverage.py provider
+
+function! s:GetCoverageFile() abort
+  " TODO(dbarnett): Limit upward search with some heuristics.
+  return fnamemodify(findfile('.coverage', ';'), ':p')
+endfunction
+
+" Polyfill for vim's pyeval().
+function! s:PyEval(expr) abort
```

Comment:
A generic polyfill like this is exactly what Maktaba is for. Consider moving this there.